docs/source/markdown/podman-top.1.md.in

```diff
@@ -9,7 +9,14 @@ podman\-top - Display the running processes of a container
 **podman container top** [*options*] *container* [*format-descriptors*]
 
 ## DESCRIPTION
-Display the running processes of the container. The *format-descriptors* are ps (1) compatible AIX format descriptors but extended to print additional information, such as the seccomp mode or the effective capabilities of a given process. The descriptors can either be passed as separated arguments or as a single comma-separated argument. Note that options and or flags of ps(1) can also be specified; in this case, Podman falls back to executing ps with the specified arguments and flags in the container.  Please use the "h*" descriptors to extract host-related information.  For instance, `podman top $name hpid huser` to display the PID and user of the processes in the host context.
+Display the running processes of the container. The *format-descriptors* are ps (1) compatible AIX format
+descriptors but extended to print additional information, such as the seccomp mode or the effective capabilities
+of a given process. The descriptors can either be passed as separated arguments or as a single comma-separated
```

Comment:
Not a hold-up for this PR, but at some point in time, an added example would be nice.

Reply:
There are examples under the EXAMPLES section

Reply:
@Luap99 perhaps I need more tea and to be more clear, but I do not see examples on this page that use a comma separated argument, just the one with the single argument `seccomp args %C`

Reply:
FWIW @Luap99 , I'm perfectly fine with not fixing with this PR.  Just something to consider for a possible future change.